benchmarks/inference-server/text-generation-inference/main.tf

```diff
@@ -18,6 +18,16 @@ locals {
 
   all_templates = concat(local.wl_templates, local.secret_templates)
 
+  hpa_cpu_template = "${path.module}/hpa-templates/hpa.cpu.yaml.tftpl"
+  hpa_custom_metric_template = "${path.module}/hpa-templates/hpa.tgi.custom_metric.yaml.tftpl"
+  tgi_podmonitoring = "${path.module}/hpa-templates/tgi-podmonitoring.yaml.tftpl"
+  custom_metrics_enabled = (
+    contains(
+      ["tgi_queue_size", "tgi_batch_current_size", "tgi_batch_current_max_tokens"],
```

Comment:
Can we set custom_metrics_enabled to false only when hpa_type is `null` or `cpu`, instead?